src/app/organizations/registerOrganization/register-organization.component.html

```diff
@@ -0,0 +1,51 @@
+<!--
+   Copyright 2019 OICR
+ *
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ *
+       http://www.apache.org/licenses/LICENSE-2.0
+ *
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<h1 mat-dialog-title>Add Organization</h1>
+<div mat-dialog-content>
+  <p>Fill out the form to send an organization request to a Dockstore curator to approve.  Once approved, the organization will be publicly visible.</p>
+  <app-alert></app-alert>
+  <form [formGroup]="registerOrganizationForm">
+    <mat-form-field>
+      <input type="text" matInput formControlName="name" placeholder="Name" required>
+      <mat-error *ngIf="name.hasError('required')">Name is required</mat-error>
+      <mat-error *ngIf="name.hasError('minlength')">Name should be at least 3 characters</mat-error>
+      <mat-error *ngIf="name.hasError('maxlength')">Name should be at most 39 characters</mat-error>
+      <mat-error *ngIf="name.hasError('pattern')">Name can only contain alphanumeric characters and start with an alphabet</mat-error>
+    </mat-form-field>
+    <mat-form-field>
+      <textarea cdkAutosizeMinRows="3" cdkTextareaAutosize type="text" matInput formControlName="description" placeholder="Description"
+        required></textarea>
+      <mat-error *ngIf="description.hasError('required')">Description is required</mat-error>
+    </mat-form-field>
+    <mat-form-field>
+      <input type="text" matInput formControlName="link" placeholder="Link to Organization Website">
+      <mat-error *ngIf="link.hasError('pattern')">Link must be a valid URL and start with http:// or https://</mat-error>
+    </mat-form-field>
+    <mat-form-field>
+      <input type="text" matInput formControlName="location" placeholder="Location">
+    </mat-form-field>
+    <mat-form-field>
+      <input type="email" matInput formControlName="contactEmail" placeholder="Contact Email Address" required>
+      <mat-error *ngIf="contactEmail.hasError('required')">Contact email address is required</mat-error>
```

Comment:
I think a contact email address might just be highly recommended as opposed to required.